src/config.js

```diff
@@ -342,3 +342,9 @@ export const CONFIG = {
     },
   },
 };
+
+export const TIME_RANGE_OPTIONS = [
+  1, 2, 7, 14, 30, 60, 90, 365,
+];
+
+export const TIME_RANGE_LIMIT = 736969;
```

Comment:
Try to write the number as a multiplication or with a variable name that makes it clear what that number represents.